help/share-and-upload-files.md

```diff
@@ -1,21 +1,103 @@
 # Share and upload files
 
-Attach files to messages, including images, documents, sound, and video.
+Zulip supports attaching multiple files to messages, including images,
+documents, sound, and video. You can edit the names of the files others see
+after you upload them.
 
-First, [open the compose box](/help/open-the-compose-box). Then
+Zulip will automatically generate a **thumbnail** for each file when you send
+the message, if it can. Image thumbnails will be shown directly in the message,
+and you can click on a thumbnail to [view the full image](/help/view-and-browse-images).
```

Comment:
This text isn't really accurate. (I see it's pre-existing from before this PR.)

The truth is we don't generate thumbnails. We have an open issue for that:
* #16210

(At one point we had an experimental implementation of generating thumbnails, and this text may have been written then. My recollection is that we never shipped that widely. In any case it's not operating now.)

We do in the message list show an image preview, at a small size, and it'd be reasonable for someone to call that piece of UI a "thumbnail". But we don't *generate* those — we just serve up the original uploaded image verbatim, and tell the browser to show it small. And because we're not doing anything to generate them, there's no "if it can"; we do the same thing for all images.

I'm not sure we need this paragraph at all. If we do want something like it, it should probably focus on the UI and not go into the backend technical details. Perhaps something like:

> For images attached to a message, a small preview will be shown directly in the message. People reading the message can click on the preview to [view the full-size image][…].